modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing a non-Red Hat vSphere CSI Operator Driver
+
+If you have installed a non-Red Hat vSphere Container Storage Interface (CSI) Operator Driver, updates for your cluster to the next major version of {product-title}, such as 4.11, may be disabled due to {product-title} 4.10 having a built-in version of the vSphere CSI Operator Driver.
```

Comment:
please replace `Operator Driver` here with just `Driver`.

Reply:
Implemented this comment.